tests/unit/s2n_openssl_test.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+#include "crypto/s2n_openssl.h"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    const char* env_libcrypto = getenv("S2N_LIBCRYPTO");
+    if (env_libcrypto == NULL) {
```

Comment:
Why would this environment variable ever be null? Just worried that we don't know if this test is actually being run or not, maybe I am overthinking though.

Reply:
It can be, depending on how you're running the tests. It's consistently set for our CI unit tests, but I don't want to break people just trying to run the unit tests locally.